src/main/java/com/hm/achievement/utils/AchievementPlaceholderHook.java

```diff
@@ -25,28 +26,62 @@ public AchievementPlaceholderHook(AdvancedAchievements plugin) {
 
 	@Override
 	public String onPlaceholderRequest(Player p, String identifier) {
+
+		int achievements = plugin.getCacheManager().getPlayerTotalAchievements(p.getUniqueId());
+
+		if (identifier.equalsIgnoreCase("achievements")) {
+			return String.valueOf(achievements);
+		}
+
+		int totalAchievements = 0;
```

Comment:
I believe you can use a small trick here instead of recomputing the total each time:
`plugin.getAchievementsAndDisplayNames().size()`

Reply:
Oh nice! :D